boards/arm/rpi_pico/rpi_pico_defconfig

```diff
@@ -0,0 +1,7 @@
+CONFIG_SOC_SERIES_RP2XXX=y
+CONFIG_SOC_RP2040=y
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=125000000
```

Comment:
The rp2040's max clock freq is 133MHz, does the chip on the pico board only run at 125MHz?
The sysclock frequency is also program modifiable at runtime.

Reply:
Yes, can't find a better source right now, hope this qualifies: https://forums.raspberrypi.com/viewtopic.php?t=302191

Reply:
looks fair enough to me... though as the last post says, you can change the rate dynamically.